sql/core/src/test/resources/sql-tests/results/pgSQL/aggregates_part4.sql.out

```diff
@@ -0,0 +1,5 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 0
```

Comment:
Thanks. Although this PR doesn't add a test coverage, we get the investigation result and have the test foundation for new JIRAs.